leapp/cli/upgrade/__init__.py

```diff
@@ -211,6 +212,11 @@ def preupgrade(args):
         report_info(get_config().get('report', 'path'), fail=True)
         sys.exit(1)
     report_info(get_config().get('report', 'path'))
+    # fetch all report messages as a list of dicts
+    messages = fetch_upgrade_report_raw(context, renderers=False)
+    report_json = get_config().get('report', 'path').replace('.txt', '.json')
```

Comment:
would about
```suggestion
    report_json = get_config().get('report', 'path_json')
```
(require update of 'config' file as well)

Reply:
Honestly I see no point in defining separate paths for txt and json report, defining a 'leapp report dir' should be enough control over output. 
But replace('.txt', '.json') can be improved